src/sentry/options/defaults.py

```diff
@@ -1784,9 +1785,30 @@
 )
 
 # Relocation
-register("relocation.enabled", default=False)
```

Comment:
I think in order to truly delete these options, we'd need to ask ops to do that? Or do we even need to do that at all 🤔

Reply:
Following comment: Does `register` change an option that already exists?

Reply:
We're not deleting them - just adding flags (ie, making them modifiable from the UI).

Register does not set options at all - it just "defines" them, so that other methods (command line, option automator, UI) have a DB target to set.